app/controllers/orders_controller.rb

```diff
@@ -15,39 +17,40 @@ def new
   end
 
   def create 
-    authorize Order
-    items = line_items_in(order_params[:cart_items_ids])
-    line_items_group_by_shop = line_items_group_by_shop(items)
-    user_address = Address.find_by(id: order_params[:user_address_id])
-
-    CreateOrdersService.call(line_items_group_by_shop, user_address)
+    @orders.each do |order| 
+      CreateGhnOrderJob.perform_later(order)
+    end
 
     flash[:success] = "Your order is created"
     redirect_to action: :index
+  end
+
+  def paypal_create_payment
+    result = PaypalClient.create_payment(orders: @orders)
+
+    if result 
+      render json: { token: result }, status: :ok
+    else 
+      render json: { error: "Payment created failed" }, status: :unprocessable_entity
+    end
+  end
+
+  def paypal_execute_payment
+    if PaypalClient.execute_payment(payment_id: params[:orderID])
+      render json: {}, status: :ok
```

Comment:
if not return anything in body, you can use
```
head :ok
```

Reply:
noted and updated